python-lib/ontology_tagger.py

```diff
@@ -26,62 +26,82 @@ def __init__(
         store_attr()
         self.matcher_dict = {}
         self.nlp_dict = {}
-
+        self.keyword_to_tag = {}
         # remove rows with missing values
         self.ontology_df.replace("", float("nan"), inplace=True)
         self.ontology_df.dropna(inplace=True)
         if self.ontology_df.empty:
             raise ValueError(
                 "No valid tags were found. Please specify at least a keyword and a tag in the ontology dataset, and re-run the recipe"
             )
+    
+    def _generate_unique_columns(self, text_df, columns):
```

Comment:
Why does this appear as a change? I thought this had been merged to main...

Could you merge main into this branch to keep the PR only about multiple languages support?